<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgWidget{
background-color: rgb(61, 56, 70);}</string>
   </property>
   <widget class="QLabel" name="welcomeLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#welcomeLabel{
font: 36pt &quot;Noto Mono&quot;;
color: rgb(255, 255, 255);
font-weight: bold;}</string>
    </property>
    <property name="text">
     <string>Welcome</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>550</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(222, 221, 218);
font: 14pt &quot;Noto Mono&quot;;</string>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordField">
    <property name="geometry">
     <rect>
      <x>263</x>
      <y>470</y>
      <width>691</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>650</y>
      <width>801</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>If it is your first time loggin into the application, enter new password which will be saved for later logins.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>710</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(237, 51, 59);</string>
    </property>
    <property name="text">
     <string>If you forget the password it will not be possible to retrieve your data! </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>670</y>
      <width>501</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>This is the master password used for encryption. It should be strong.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>730</y>
      <width>1021</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>If you forgot the password you can delete the database in ~/.password_manager/password_manager.sqlite3 in order to use the application again.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>768</y>
      <width>521</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(237, 51, 59);</string>
    </property>
    <property name="text">
     <string>Author of this application does not ensure the security of stored data.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Mono&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>This is a local Password Manager application. </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Mono&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>It uses AES algorithm for data encryption and SQLite database.</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordError">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>520</y>
      <width>691</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(237, 51, 59);
font: 9pt &quot;Noto Mono&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>passwordField</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>bgWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>passwordField</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>608</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>585</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
